manifests/c/CHMS/Behindyou/1.0.0/CHMS.Behindyou.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CHMS.Behindyou
+PackageVersion: 1.0.0

```

```suggestion
PackageVersion: 1.5
```
This must match Control Panel.